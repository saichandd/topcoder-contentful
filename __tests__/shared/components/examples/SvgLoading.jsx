import React from 'react';
import renderer from 'react-test-renderer';

import SvgLoading from 'components/examples/SvgLoading';

test('matches snapshots', () => {
  const cmp = renderer.create(<SvgLoading />);
  expect(cmp.toJSON()).toMatchSnapshot();
});
