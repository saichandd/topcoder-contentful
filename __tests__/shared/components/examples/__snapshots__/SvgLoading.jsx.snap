// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<div>
  <h1>
    SVG Loading
  </h1>
  <p>
    This component show how to load
     
    <code>
      .svg
    </code>
     
    assets with use of
    <code>
       
      babel-plugin-inline-react-svg
    </code>
    .
  </p>
  <svg
    height="200px"
    viewBox="0 0 283.46455 85.039373"
    width="640px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M12.906 12.034h260.215v60.327H12.907z"
      fill="none"
      stroke="green"
      strokeWidth=".49"
    />
    <text
      fontFamily="sans-serif"
      fontSize="20"
      fontWeight="400"
      letterSpacing="0"
      style={
        Object {
          "lineHeight": "125%",
        }
      }
      transform="translate(-164.279 -116.897)"
      wordSpacing="0"
      x="196.071"
      y="141.291"
    >
      <tspan
        x="196.071"
        y="166.291"
      >
        A Sample SVG asset :)
      </tspan>
    </text>
  </svg>
</div>
`;
